### ISSUE TITLE
Content wrongly loses its heading after it is copied and pasted

### ISSUE THREAD

**Issue by Softvision-MariusComan** [mozilla/notes#1181]
**[Affected versions]:**
- Firefox Release 60.0.2 and above

**[Affected Platforms]:**
- All Windows
- All Linux
- All Mac

**[Prerequisites]:**
- Have a new Firefox profile with the latest version of the "Firefox Notes" add-on (4.2.0dev custom built on 20.06.2018) installed.

**[Steps to reproduce]:**
1. Open the browser with the profile from prerequisites.
2. Click the "New Note" button and select any heading except "Body".
3. Write a string and select it by mouse drag.
4. Copy the selected string.
5. Press the "Enter" key and paste the previously copied text.
6. Observe the heading of the pasted string.

**[Expected result]:**
- The pasted string has the same heading as the one from step 3.

**[Actual result]:**
- The heading is reset to "Body".

**[Notes]:**
- The issue is not reproducible if the content is selected using the "Ctrl+A" shortcut.
- The style formatting options (such as bold, italic, strikethrough and underline) are not lost at paste.
- Attached a screen recording of the issue: 
![rec of the issue](https://user-images.githubusercontent.com/31243259/41725917-a9f4630c-7579-11e8-8ff0-7755befe7152.gif)

**Comment by vladikoff** [comment]
This seems like ckeditor behaviour, do you think it is worth filing an issue on this on their end @Softvision-MariusComan ? https://ckeditor5.github.io/ seems to reproduce this

**Comment by Softvision-MariusComan** [comment]
Yes, I think it's worth filing an issue on their end. We should also keep this one open in case we need to dupe others to it.
